src/sdk/object_sdk.js

```diff
@@ -207,19 +207,16 @@ class ObjectSDK {
     async load_requesting_account(req) {
         try {
             const token = this.get_auth_token();
-            if (!token) return;
+            if (!token && !process.env.NC_NSFS_NO_DB_ENV) return;
```

Comment:
How about we add a bucketspace method that will decide if to allow anonymous requests?
I think this will encapsulate this check for env NC_NSFS_NO_DB_ENV which is quite weird in this context.